src/modules/appUI.js

```diff
@@ -34,4 +73,4 @@ const displaySeries = () => {
   });
 };
```

Comment:
Let's also add the popup modal for the series in order to complete the popup task. 👍